tests/framework-tests/preferences/js/PanelsTests.js

```diff
@@ -1239,10 +1255,8 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 expect: 2,
                 name: "Test the rendering of the text size panel",
                 sequence: [{
-                    func: "{textSize}.refreshView"
-                }, {
                     listener: "fluid.tests.testDefault",
-                    event: "{textSize}.events.afterRender"
+                    event: "{textSizePanel textfieldSlider}.events.onCreate"
```

Comment:
Why do we run tests for the text size panel at the creation of its subcomponent "textfieldSlider"? If it's because the onCreate event of "textfieldSlider" component sometimes gets fired after the afterRender event of "textSizePanel" component, it might make more sense to run tests at an aggregate event of textfieldSlider.onCreate + textSizePanel.afterRender to make sure both slider and panel are ready when tests run.